services/api/src/api/utils.py

```diff
@@ -59,6 +60,10 @@ class UnexpectedError(ApiCustomError):
 
     def __init__(self, message: str, cause: Optional[BaseException] = None):
         super().__init__(message, HTTPStatus.INTERNAL_SERVER_ERROR, "UnexpectedError", cause)
+        if cause:
+            logging.exception(message, exc_info=cause)
+        else:
+            logging.exception(message)
```

Comment:
Note that basically: `logging.exception = logging.error(..., exc_info=True)`
```suggestion
        logging.error(message, exc_info=cause)
```

Reply:
https://github.com/python/cpython/blob/5f0b81905084eb5da7131d06a6a4f15beeab460f/Lib/logging/__init__.py#L2117

Reply:
OHHH. It's perfect